src/content/docs/en/guides/markdown-content.mdx

```diff
@@ -355,6 +355,12 @@ export default defineConfig({
       // Note: Shiki has countless langs built-in, including .astro!
       // https://shiki.style/languages
       langs: [],
+      // Add custom aliases for languages
+      // The key is the alias you'll use in your markdown files and the value is the language ID used by Shiki
```

Comment:
```suggestion
      // Map an alias to a Shiki language ID: https://shiki.style/languages#bundled-languages
```
I think this might be sufficient? Especially with the link below!

I do wonder about providing the languages link specifically, though. Because people might not realize that `js`, which everyone uses all the time, is itself an alias for `javascript` (the ID). So it might be helpful for them to have that list of language IDs!